app/components/auth/LogoutButton.tsx

```
@@ -0,0 +1,135 @@
+import React, { useState } from 'react';
+import { TouchableOpacity, Text, StyleSheet, Alert, ActivityIndicator, View } from 'react-native';
+import { useAuth } from '@/src/context/AuthContext';
+import { Ionicons } from '@expo/vector-icons';
+
+import { TouchableOpacity, Text, StyleSheet, Alert, ActivityIndicator, View, ViewStyle, StyleProp } from 'react-native';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'TouchableOpacity'. Consider to delete it or rename it.

'TouchableOpacity' is defined here:


(lint/suspicious/noRedeclare)